src/bid/errors.rs

```diff
@@ -1,6 +1,6 @@
 //! Errors related to the Bid Generation
 
-use dusk_plonk::jubjub::Fr as JubJubScalar;
+use dusk_plonk::prelude::JubJubScalar;
```

Comment:
We shouldn't import from prelude unless we're using everything

Reply:
Done